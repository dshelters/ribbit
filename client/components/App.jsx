import React from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';
import Navbar from './Navbar';
import Home from './homepage/Home';
import Room from './room/Room';

const App = () => (
  <div>
    <p>Hello David!</p>
    <Navbar />
    <Route exact path="/" component={Home} />
    <Route path="/room" component={Room} />
  </div>
);

export default App;

