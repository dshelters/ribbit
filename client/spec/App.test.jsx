import React from 'react';
import renderer from 'react-test-renderer';
import App from '../components/App';

test('App component snapshot test', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
