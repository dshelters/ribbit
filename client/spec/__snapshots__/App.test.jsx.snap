// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component snapshot test 1`] = `
<div>
  <p>
    Hello David!
  </p>
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      This is the navbar with logo
    </a>
  </div>
</div>
`;
